docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,10 +383,39 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file`` or passing queries to
+the standard input uses the batch mode (non-interactive), in which
+the Trino CLI exits after executing all queries, does not report progress,
+and prints results in the ``CSV`` format.
+
+The following options are available to further configure the CLI in batch
+mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--execute=<execute>``
+    - Execute specified statements and exit.
```

Comment:
can you do multiple statements actually?

Also ... should we not reword to process the specified queries ..

Reply:
Technically we should be calling these statements. Not every SQL statement returns a result and hence is not a query.

Reply:
@hashhar is right and I did not reword this, just copied from the output of `--help`. Being able to execute multiple statements is one of the coolest things about the CLI client.

Reply:
ok .. maybe we want to clarify more what happens but in any case.. this is good to go for starters..